src/Tribe/Views/V2/Assets.php

```diff
@@ -66,6 +66,19 @@ public function register() {
 			]
 		);
 
+		tribe_asset(
+			$plugin,
+			'tribe-events-views-v2-multiday-events',
+			'views/multiday-events.js',
+			[ 'jquery', 'tribe-common', 'tribe-events-views-v2-manager' ],
+			'wp_enqueue_scripts',
+			[
+				'priority'     => 10,
+				'conditionals' => [ $this, 'should_enqueue_frontend' ],
```

Comment:
So far this should only be included in month view. Let's see if it's worth it to do it now or leave a to-do to revisit this in the future (probably when week view is implemented).

Reply:
It needs to be included on all views.